src/web/server/htmlTemplate.ts

```diff
@@ -37,13 +45,35 @@ export const htmlTemplate = ({
             ? 'favicon-32x32.ico'
             : 'favicon-32x32-dev-yellow.ico';
 
+    // ********************************
+    // ****** high priority script ****
+    // ********************************
     const priorityScriptTags = priorityScripts.map(
         src => `<script defer src="${src}"></script>`,
     );
+    // transpiled with preset-env
+    const priorityLegacyScriptTags = priorityLegacyScripts.map(
+        src => `<script defer nomodule src="${src}"></script>`,
+    );
+    // transpiled with preset-modules
+    const priorityNonLegacyScriptTags = priorityNonLegacyScripts.map(
```

Comment:
Don't these two *replace* `priorityScriptTags` and `lowPriorityScriptTags`?

Reply:
I may have made the wrong assumption, but here is my reasoning; we have 3 cases for loading scripts:

- I do not care about browser what the browser supports
- I care if the browser can support `type="module"`
- I care if the browser cannot support `type="module"`

Adding code into the 1st case means it will be loaded by browsers that support `type="module"` which is something we want to avoid

Reply:
Yup, sorry, read that wrong, carry on!